src/System.Private.Xml/src/System/Xml/Serialization/XmlSchemaImporter.cs

```diff
@@ -418,66 +408,7 @@ private TypeMapping ImportElementType(XmlSchemaElement element, string identifie
 
         private void RunSchemaExtensions(TypeMapping mapping, XmlQualifiedName qname, XmlSchemaType type, XmlSchemaObject context, TypeFlags flags)
         {
-            string typeName = null;
```

Comment:
adding #if everywhere CodeDom was used would make the code dirty, so I decided to remove the code instead. I believe I only kept one piece behind #if CODEDOM, I will remove that one as well to keep the consistency.